tests/zfs-tests/tests/functional/cli_root/zpool_resilver/zpool_resilver_restart.ksh

```diff
@@ -40,18 +40,12 @@
 #	   deferred
 #	4. Manually restart the resilver with all drives
 #
-# NOTES:
-#	Artificially limit the scrub speed by setting the zfs_scan_vdev_limit
-#	low and adding a 50ms zio delay in order to ensure that the resilver
-#	does not complete early.
-#
 
 verify_runnable "global"
 
 function cleanup
 {
-	log_must zinject -c all
-	log_must set_tunable64 zfs_scan_vdev_limit $ZFS_SCAN_VDEV_LIMIT_DEFAULT
```

Comment:
`ZFS_SCAN_VDEV_LIMIT_{DEFAULT|SLOW}` should be removed from `zpool_resilver.cfg`.